src/samples/audio/detect_test.c

```diff
@@ -310,6 +318,18 @@ static struct comp_dev *test_keyword_new(const struct comp_driver *drv,
 		goto fail;
 	}
 
+	/* global buffer where we collect the 990 ms of data
+	 * as required by the algorithm
+	 */
+	cd->input = rballoc_align(0, SOF_MEM_CAPS_RAM,
+			sizeof(int16_t) * KWD_NN_IN_BUFF_SIZE, 64);
```

Comment:
Can 64 be put in a macro, such as KWD_NN_BUFF_ALIGN or something?

Reply:
Ok. But I have alignments also in src/samples/audio/kwd_nn_detect_test.c:
static __aligned(64) uint8_t preprocessed_data[KWD_NN_CONFIG_PREPROCESSED_SIZE];
static __aligned(8) uint8_t confidences[KWD_NN_CONFIDENCES_SIZE];

Should I define in each source a different macro?

Reply:
In the __aligned you can put raw numbers, that's fine. In there it's _obviously_ an alignment. However for `rballoc_align` it isn't immediately obvious that the final parameter is an alignment so use a macro that explains that it is an alignment.